### ISSUE TITLE
paddlex PP-LCNet_x1_0_doc_ori.yaml 训练估计部分怎么一直都是0呢？

### ISSUE THREAD

**Issue by cqray1990** [PaddlePaddle/PaddleX#4966]
PP-LCNet_x1_0_doc_ori.yaml 配置文件内容：

                   # global configs
                Global:
                  checkpoints: null
                  pretrained_model: null
                  output_dir: ./output/
                  device: gpu
                  save_interval: 10
                  eval_during_train: True
                  eval_interval: 10
                  epochs: 60
                  print_batch_step: 10
                  use_visualdl: False
                  # used for static mode and model export
                  image_shape: [3, 224, 224]
                  save_inference_dir: ./inference
                
                # mixed precision
                AMP:
                  use_amp: False
                  use_fp16_test: False
                  scale_loss: 128.0
                  use_dynamic_loss_scaling: True
                  use_promote: False
                  # O1: mixed fp16, O2: pure fp16
                  level: O1
                
                # model architecture
                Arch:
                  name: PPLCNet_x1_0
                  class_num: 4
                  pretrained: True
                
                # loss function config for traing/eval process
                Loss:
                  Train:
                    - CELoss:
                        weight: 1.0
                        epsilon: 0.1
                  Eval:
                    - CELoss:
                        weight: 1.0
                
                
                Optimizer:
                  name: Momentum
                  momentum: 0.9
                  lr:
                    name: Cosine
                    learning_rate: 0.4
                    warmup_epoch: 5
                  regularizer:
                    name: 'L2'
                    coeff: 0.00003
                
                
                # data loader for train and eval
                DataLoader:
                  Train:
                    dataset:
                      name: ImageNetDataset
                      image_root: ./dataset/text_image_orientation/
                      cls_label_path: ./dataset/text_image_orientation/train_list.txt
                      delimiter: "\t"
                      transform_ops:
                        - DecodeImage:
                            to_rgb: True
                            channel_first: False
                        - RandCropImage:
                            size: 224
                        - TimmAutoAugment:
                            prob: 0.0
                            config_str: rand-m9-mstd0.5-inc1
                            interpolation: bicubic
                            img_size: 224
                        - NormalizeImage:
                            scale: 1.0/255.0
                            mean: [0.485, 0.456, 0.406]
                            std: [0.229, 0.224, 0.225]
                            order: ''
                        - RandomErasing:
                            EPSILON: 0.0
                            sl: 0.02
                            sh: 1.0/3.0
                            r1: 0.3
                            attempt: 10
                            use_log_aspect: True
                            mode: pixel
                    sampler:
                      name: DistributedBatchSampler
                      batch_size: 256
                      drop_last: False
                      shuffle: True
                    loader:
                      num_workers: 8
                      use_shared_memory: True
                
                  Eval:
                    dataset: 
                      name: ImageNetDataset
                      image_root: ./dataset/text_image_orientation/
                      cls_label_path: ./dataset/text_image_orientation/test_list.txt
                      delimiter: "\t"
                      transform_ops:
                        - DecodeImage:
                            to_rgb: True
                            channel_first: False
                        - ResizeImage:
                            resize_short: 256
                        - CropImage:
                            size: 224
                        - NormalizeImage:
                            scale: 1.0/255.0
                            mean: [0.485, 0.456, 0.406]
                            std: [0.229, 0.224, 0.225]
                            order: ''
                    sampler:
                      name: DistributedBatchSampler
                      batch_size: 64
                      drop_last: False
                      shuffle: False
                    loader:
                      num_workers: 4
                      use_shared_memory: True
                
                Infer:
                  infer_imgs: ./test_img/
                  batch_size: 1
                  transforms:
                    - DecodeImage:
                        to_rgb: True
                        channel_first: False
                    - ResizeImage:
                        resize_short: 256
                    - CropImage:
                        size: 224
                    - NormalizeImage:
                        scale: 1.0/255.0
                        mean: [0.485, 0.456, 0.406]
                        std: [0.229, 0.224, 0.225]
                        order: ''
                    - ToCHWImage:
                  PostProcess:
                    name: Topk
                    topk: 1
                    class_id_map_file: ppcls/utils/PULC_label_list/text_image_orientation_label_list.txt
                
                Metric:
                  Train:
                    - TopkAcc:
                        topk: [1]
                  Eval:
                    - TopkAcc:
                        topk: [1]
                

[2026/01/31 09:41:36] ppcls INFO: [Train][Epoch 277/600][Iter: 600/604]lr(LinearWarmup): 0.05771145, top1: 0.95470, CELoss: 0.42807, loss: 0.42807, batch_cost: 0.26462s, reader_cost: 0.00006, ips: 725.57739 samples/s, eta: 14:20:25
[2026/01/31 09:41:39] ppcls INFO: [Train][Epoch 277/600][Avg]top1: 0.95469, CELoss: 0.42808, loss: 0.42808
[2026/01/31 09:41:47] ppcls INFO: [Eval][Epoch 277][Avg], top1: 0.00000
[2026/01/31 09:41:47] ppcls INFO: [Eval][Epoch 277][best metric: 0]
[2026/01/31 09:41:47] ppcls INFO: Already save model in PaddleX/output_cls/best_model/best_model

**Comment by FreemanTang** [comment]
> PP-LCNet_x1_0_doc_ori.yaml 配置文件内容：
> 
> ```
>                # global configs
>             Global:
>               checkpoints: null
>               pretrained_model: null
>               output_dir: ./output/
>               device: gpu
>               save_interval: 10
>               eval_during_train: True
>               eval_interval: 10
>               epochs: 60
>               print_batch_step: 10
>               use_visualdl: False
>               # used for static mode and model export
>               image_shape: [3, 224, 224]
>               save_inference_dir: ./inference
>             
>             # mixed precision
>             AMP:
>               use_amp: False
>               use_fp16_test: False
>               scale_loss: 128.0
>               use_dynamic_loss_scaling: True
>               use_promote: False
>               # O1: mixed fp16, O2: pure fp16
>               level: O1
>             
>             # model architecture
>             Arch:
>               name: PPLCNet_x1_0
>               class_num: 4
>               pretrained: True
>             
>             # loss function config for traing/eval process
>             Loss:
>               Train:
>                 - CELoss:
>                     weight: 1.0
>                     epsilon: 0.1
>               Eval:
>                 - CELoss:
>                     weight: 1.0
>             
>             
>             Optimizer:
>               name: Momentum
>               momentum: 0.9
>               lr:
>                 name: Cosine
>                 learning_rate: 0.4
>                 warmup_epoch: 5
>               regularizer:
>                 name: 'L2'
>                 coeff: 0.00003
>             
>             
>             # data loader for train and eval
>             DataLoader:
>               Train:
>                 dataset:
>                   name: ImageNetDataset
>                   image_root: ./dataset/text_image_orientation/
>                   cls_label_path: ./dataset/text_image_orientation/train_list.txt
>                   delimiter: "\t"
>                   transform_ops:
>                     - DecodeImage:
>                         to_rgb: True
>                         channel_first: False
>                     - RandCropImage:
>                         size: 224
>                     - TimmAutoAugment:
>                         prob: 0.0
>                         config_str: rand-m9-mstd0.5-inc1
>                         interpolation: bicubic
>                         img_size: 224
>                     - NormalizeImage:
>                         scale: 1.0/255.0
>                         mean: [0.485, 0.456, 0.406]
>                         std: [0.229, 0.224, 0.225]
>                         order: ''
>                     - RandomErasing:
>                         EPSILON: 0.0
>                         sl: 0.02
>                         sh: 1.0/3.0
>                         r1: 0.3
>                         attempt: 10
>                         use_log_aspect: True
>                         mode: pixel
>                 sampler:
>                   name: DistributedBatchSampler
>                   batch_size: 256
>                   drop_last: False
>                   shuffle: True
>                 loader:
>                   num_workers: 8
>                   use_shared_memory: True
>             
>               Eval:
>                 dataset: 
>                   name: ImageNetDataset
>                   image_root: ./dataset/text_image_orientation/
>                   cls_label_path: ./dataset/text_image_orientation/test_list.txt
>                   delimiter: "\t"
>                   transform_ops:
>                     - DecodeImage:
>                         to_rgb: True
>                         channel_first: False
>                     - ResizeImage:
>                         resize_short: 256
>                     - CropImage:
>                         size: 224
>                     - NormalizeImage:
>                         scale: 1.0/255.0
>                         mean: [0.485, 0.456, 0.406]
>                         std: [0.229, 0.224, 0.225]
>                         order: ''
>                 sampler:
>                   name: DistributedBatchSampler
>                   batch_size: 64
>                   drop_last: False
>                   shuffle: False
>                 loader:
>                   num_workers: 4
>                   use_shared_memory: True
>             
>             Infer:
>               infer_imgs: ./test_img/
>               batch_size: 1
>               transforms:
>                 - DecodeImage:
>                     to_rgb: True
>                     channel_first: False
>                 - ResizeImage:
>                     resize_short: 256
>                 - CropImage:
>                     size: 224
>                 - NormalizeImage:
>                     scale: 1.0/255.0
>                     mean: [0.485, 0.456, 0.406]
>                     std: [0.229, 0.224, 0.225]
>                     order: ''
>                 - ToCHWImage:
>               PostProcess:
>                 name: Topk
>                 topk: 1
>                 class_id_map_file: ppcls/utils/PULC_label_list/text_image_orientation_label_list.txt
>             
>             Metric:
>               Train:
>                 - TopkAcc:
>                     topk: [1]
>               Eval:
>                 - TopkAcc:
>                     topk: [1]
> ```
> 
> [2026/01/31 09:41:36] ppcls INFO: [Train][Epoch 277/600][Iter: 600/604]lr(LinearWarmup): 0.05771145, top1: 0.95470, CELoss: 0.42807, loss: 0.42807, batch_cost: 0.26462s, reader_cost: 0.00006, ips: 725.57739 samples/s, eta: 14:20:25 [2026/01/31 09:41:39] ppcls INFO: [Train][Epoch 277/600][Avg]top1: 0.95469, CELoss: 0.42808, loss: 0.42808 [2026/01/31 09:41:47] ppcls INFO: [Eval][Epoch 277][Avg], top1: 0.00000 [2026/01/31 09:41:47] ppcls INFO: [Eval][Epoch 277][best metric: 0] [2026/01/31 09:41:47] ppcls INFO: Already save model in PaddleX/output_cls/best_model/best_model

@cqray1990 大佬，你好，请问你有找到Eval为0的具体原因吗？
```
ppcls INFO: [Train][Epoch 1/480][Avg]top1: 0.63121, CELoss: 0.72477, loss: 0.72477
ppcls INFO: [Eval][Epoch 1][Avg], top1: 0.00000
ppcls INFO: [Eval][Epoch 1][best metric: 0]
```
